src/multidim/box.jl

```diff
@@ -0,0 +1,34 @@
+# Multidimensional intervals, called IntervalBox
+
+#using ValidatedNumerics
+#using FixedSizeArrays
+
+import Base:
+    ⊆, ∩, isempty
+
+export IntervalBox
```

Comment:
`IntervalBox` is exported in src/ValidatedNumerics.jl; I don't think it needs to be exported here.